envs/example/defaults.yml

```diff
@@ -107,8 +107,11 @@ nova:
     debug: True
     verbose: True
 
+ceph:
+  enabled: false
+
 cinder:
-  enabled: True
+  enabled: {{ ceph.enabled }}
   volume_type: file
```

Comment:
We need to continue to support existing installations where are using the Nimble and file backends, so cinder should still function w/o ceph.